### PR TITLE
Added pwned email check module and prestashop CMS detection util function 

### PR DESCRIPTION
**Pull request by vikas-kundu**
The previous pull request didn't contain my module so closed it and made this one. Sorry for the mistake, i issues pull without adding the necessary changes to my branch. This is my first ever pull request.

### CODE REVIEW

**Comment by saeeddhqan** [comment]
Can you show me a gif?

**Comment by vikas-kundu** [comment]
@saeeddhqan here is the gif: https://asciinema.org/a/JE4S9MhFJxFQRbUAngvIq82ZO

**Comment by saeeddhqan** [comment]
I'll add it by the new version.

**Comment by vikas-kundu** [comment]
@saeeddhqan i have also added a prestashop detection function in the core util of cms_identify.py. Here is a gif of the same:
[https://asciinema.org/a/4vAlrQ1CFRUsnBrZeGjeTmR9f](https://asciinema.org/a/4vAlrQ1CFRUsnBrZeGjeTmR9f)

**Comment by saeeddhqan** [comment]
Work on the new module style.

**Comment by vikas-kundu** [comment]
> Work on the new module style.

can you please elaborate a bit more on the aspects that i need to work?

**Comment by saeeddhqan** [comment]
@z3y50n will help you.

**Comment by z3y50n** [comment]
Since version 2.0 there is a new module structure that can be found [here](https://github.com/saeeddhqan/Maryam/wiki/Development-Guide#make-a-module)

Feel free to check out other modules as well to get a better understanding!

**Comment by vikas-kundu** [comment]
@saeeddhqan i have made the necessary style changes:
[https://github.com/saeeddhqan/Maryam/pull/79/commits/b9be0ca0d1cf353a2a5cb365d0d8e16024c4274f](https://github.com/saeeddhqan/Maryam/pull/79/commits/b9be0ca0d1cf353a2a5cb365d0d8e16024c4274f)
Here is a gif of the same:
https://asciinema.org/a/glsomB5y9m6tePdsUk6vJX6SX

File: modules/osint/email_pwned.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		return result.json()
+	else:
+		return False
+
+
+def module_api(self):
+	output, pwns = {}, {}
+	email = self.options["email"]
+
+	if(re.search('^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$', email)):
```
Comment (saeeddhqan) [inline modules/osint/email_pwned.py]: Remove parentheses.
Insert r before regex.
Use double-quotes for regexes.
Use single-quotes for simple strings such as 'email'

File: modules/osint/email_pwned.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		self.verbose('Searching for pwning...')
```
Comment (saeeddhqan) [inline modules/osint/email_pwned.py]: self.verbose('[PWNED] ..

File: modules/osint/email_pwned.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		pwns = scrap(email)
+		if pwns:
+			output["Breaches"] = [{"BreachName": i["Name"], "BreachDomain":i["Domain"]} for i in pwns["Breaches"]]
```
Comment (saeeddhqan) [inline modules/osint/email_pwned.py]: Use single-quotes.

File: modules/osint/email_pwned.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+			if pwns["Pastes"]:
+				output["Pastes"] = [{"PasteId": j["Id"], "PasteSource": j["Source"]} for j in pwns["Pastes"]]
+			else:
+				output["Pastes"] = "Pastes not available for this email"
+		else:
+			output["Pastes"] = output["Breaches"] = "Email not pwned"
+	else:
+		self.error("Invalid Email")
```
Comment (saeeddhqan) [inline modules/osint/email_pwned.py]: It won't print.

File: modules/osint/email_pwned.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+	),
+	"examples": ("pwned -e <email> --output",)
+}
+
+
+def scrap(email):
+	url = "https://haveibeenpwned.com/unifiedsearch/"+email
+	scraper = cloudscraper.create_scraper()
+	result = scraper.get(url)
+	if(result.text != ""):
```
Comment (saeeddhqan) [inline modules/osint/email_pwned.py]: Remove parentheses.
Instead of "" write ''

File: modules/osint/email_pwned.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+	"sources": (["https://www.haveibeenpwned.com"]),
+	"options": (
+		("email", None, True, "Email to search for breach", "-e", "store", str),
+	),
+	"examples": ("pwned -e <email> --output",)
+}
+
+
+def scrap(email):
+	url = "https://haveibeenpwned.com/unifiedsearch/"+email
```
Comment (saeeddhqan) [inline modules/osint/email_pwned.py]: Use f"...{email}"

**Comment by vikas-kundu** [comment]
@saeeddhqan i have made the changes as you asked [https://github.com/saeeddhqan/Maryam/pull/79/commits/d70a178b3ca70fe50c7ba1d2ddf3f99d4da5a8fd](https://github.com/saeeddhqan/Maryam/pull/79/commits/d70a178b3ca70fe50c7ba1d2ddf3f99d4da5a8fd)

**Comment by saeeddhqan** [comment]
@z3y50n will merge it.

**Comment by vikas-kundu** [comment]
@saeeddhqan is there a slack channel or something that i can join?

**Comment by saeeddhqan** [comment]
#owasp-maryam.
If you couldn't find it, please give me your email address.

**Comment by vikas-kundu** [comment]
Thank you. I found it and joined.

**Pull request merged**